udf/bits_to_last_seen_int.sql

```diff
@@ -0,0 +1,41 @@
+/*
+Given a BYTE, return the number of days since the client was
+last seen.
+
+If no bits are set, returns NULL, indicating we don't know.
+Otherwise the results are 0-indexed, meaning \x01 will return 0.
+
+Tests showed this being 5-10x faster than the simpler alternative:
+CREATE TEMP FUNCTION
+  udf_bits_to_last_seen_int(b BYTES) AS ((
+    SELECT MIN(n)
+    FROM UNNEST(GENERATE_ARRAY(0, 364)) AS n
+    WHERE BIT_COUNT(SUBSTR(b >> n, -1) & b'\x01') > 0));
+
+See also: bits_to_last_seen_int.sql
+*/
+
+CREATE TEMP FUNCTION
+  udf_bits_to_last_seen_int(b BYTES) AS ((
```

Comment:
How about `udf_bits_to_days_since_seen`? That would feel more consistent to me with existing terminology. We have days_seen_bits which are used to calculate days_since_X columns in clients_last_seen.

`last_seen_int` sounds to me like we're returning the integer which was last seen, rather than returning an integer representing the last seen occurrence of something.

Reply:
Agreed, it is confusing. How about `udf_bits_to_days_since_last_seen` and `udf_bits_to_days_since_first_seen`

Reply:
I would argue that the standard we have is `days_since_<action>` and that `seen` is the default action that means most recent occurrence rather than `last_seen`.

So `udf_bits_to_days_since_first_seen` sounds good to me, but I'd prefer `udf_bits_to_days_since_seen` over `udf_bits_to_days_since_last_seen`